spec/models/custom_attribute_spec.rb

```diff
@@ -0,0 +1,19 @@
+describe CustomAttribute do
+  it "return correct string field type" do
+    string_custom_attribute = FactoryGirl.build(:custom_attribute, :name => "foo", :value => "string", :resource_type => 'ExtManagementSystem')
```

Comment:
I am using `let` everytime as it is possible, I thought that it is particular for better reading of specs and when somebody will add new spec he can reuse it.
http://betterspecs.org/#let - but your point of view is also understandable.

But @alongoldboim changed it already, so I will leave it as it is now, ok ?